crates/bevy_ecs/src/bundle.rs

```diff
@@ -8,13 +8,36 @@ use crate::{
 use bevy_ecs_macros::all_tuples;
 use std::{any::TypeId, collections::HashMap};
 
-/// An ordered collection of components
+/// An ordered collection of components, commonly used for spawning entities, and adding and removing components in bulk.
 ///
-/// See [Bundle]
-/// # Safety
-/// [Bundle::type_info] must return the TypeInfo for each component type in the bundle, in the _exact_
-/// order that [Bundle::get_components] is called.
-/// [Bundle::from_components] must call `func` exactly once for each [TypeInfo] returned by [Bundle::type_info]
+///You cannot query for a bundle, only individual components within it.
+///
+///Typically, you will simply use `#[derive(Bundle)]` when creating your own `Bundle`.
+///The `Bundle` trait is automatically implemented for tuples of components:
+///`(ComponentA, ComponentB)` is a very convenient shorthand when working with one-off collections of components.
+///Note that both `()` and `(ComponentA, )` are valid tuples.
+///
+///You can nest bundles like so:
+///```
+///use bevy_ecs::bundle::Bundle;
```

Comment:
as this is not the import user should use, could you hide it ? with `/// # use...`

Reply:
That's a useful trick. Added :)